### PR TITLE
Added 101 tutorial to 1.9 documentation

### PR DESCRIPTION
**Pull request by joerg84**
## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing).

### CODE REVIEW

**Comment by joerg84** [comment]
Same content as for 1.8, updated links.

File: 1.9/usage/tutorials/dcos-101/cli.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+# Outcome
+Congratulations! We have successfully connected to your cluster using the DC/OS CLI, and started exploring some of the CLI commands.
+We will make further use of the CLI in the sections that follow.
+
+# Deep Dive
+We have already encountered several DC/OS components (including Mesos, Marathon, or Metronome) while experimenting with the DC/OS CLI.
+But what other components make up DC/OS?
+
+## DC/OS components
+DC/OS includes a number of components, many of which you can safely ignore for now and hence we only list the most relevant components here.
```
Comment (joel-hamill) [inline 1.9/usage/tutorials/dcos-101/cli.md]: `Here are the DC/OS components that are relevant to this tutorial. A full description of all components...`

File: 1.9/usage/tutorials/dcos-101/cli.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+A description of all components can be found in the [documentation](https://dcos.io/docs/1.9/overview/components/).
+* Marathon starts and monitors DC/OS applications and services.
+* Mesos is the kernel of DC/OS and responsible for low-level task maintenance.
+* Mesos DNS provides service discovery within the cluster.
+* Minuteman is the internal layer 4 load balancer.
```
Comment (joel-hamill) [inline 1.9/usage/tutorials/dcos-101/cli.md]: Minuteman is now part of navstar in 1.9

File: 1.9/usage/tutorials/dcos-101/cli.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+* Admin Router is an open source NGINX configuration that provides central authentication and proxy to DC/OS services.
+* Universe is the package repository that holds the DC/OS services (e.g., Apache Spark of Apache Cassandra) that you can install on your cluster directly from the DC/OS UI and CLI.
+
+## DC/OS Terminology
+Again we will focus on the most important terminology here. More details can be found in the [documentation](https://dcos.io/docs/1.9/overview/architecture/).
```
Comment (joel-hamill) [inline 1.9/usage/tutorials/dcos-101/cli.md]: Remove line 50 entirely.

File: 1.9/usage/tutorials/dcos-101/index.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+---
+post_title: DC/OS 101
+menu_order: 1
+---
+
+This DC/OS 101 tutorial will guide you through the steps necessary to get started using DC/OS.
+
+The tutorial is designed to provide users and applications developers an overview of the basic pieces of DC/OS. We will discuss basic recipes for developing and orchestrating applications for a distributed landscape. In order to provide hands-on experience, we will develop and deploy multiple applications (including stateful, dockerized, and non-dockerized) on an actual cluster during each step of the tutorial. To provide even more rationale and background information the tutorial also includes optional deep dives into relevant topics.
```
Comment (joel-hamill) [inline 1.9/usage/tutorials/dcos-101/index.md]: `...overview of the DC/OS basics.`

File: 1.9/usage/tutorials/dcos-101/loadbalancing.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
+---
+post_title: Load-balancing
+nav_title: Load-balancing
+menu_order: 7
+---
+
+# Prerequisites
+* A [running DC/OS cluster](/docs/1.9/usage/tutorials/dcos-101/cli/) with [the DC/OS CLI installed](link to installation instructions].
```
Comment (joel-hamill) [inline 1.9/usage/tutorials/dcos-101/loadbalancing.md]: Add link here `(link to installation instructions]`

File: 1.9/usage/tutorials/dcos-101/loadbalancing.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+* [app2 and Marathon-LB](/docs/1.9/usage/tutorials/dcos-101/app2/) deployed and running in your cluster.
+
+# Objective
+In this final session, we will scale our application to multiple instances and learn how internal and external services choose which instance to use once the application has been scaled.
+
+# Steps
+Load-balancers decide which instance of an app internal or external services should use. With DC/OS, you have two different built-in load-balancer options: : [Marathon-LB](/docs/1.9/usage/service-discovery/marathon-lb/) and [Named VIPs](/docs/1.9/usage/service-discovery/load-balancing-vips/).
```
Comment (joel-hamill) [inline 1.9/usage/tutorials/dcos-101/loadbalancing.md]: remove double-colons `options: :`

File: 1.9/usage/tutorials/dcos-101/loadbalancing.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+We have seen both before in the context of service discovery and especially marathon-lb when making app2 publically available.
```
Comment (joel-hamill) [inline 1.9/usage/tutorials/dcos-101/loadbalancing.md]: global terminology change: `Marathon-LB`

Comment (joel-hamill) [inline 1.9/usage/tutorials/dcos-101/loadbalancing.md]: `We have already explored these load balancers in the context of service discovery, when we used Marathon-LB to publicly expose app2. Now let's explore them a bit more.`

File: 1.9/usage/tutorials/dcos-101/loadbalancing.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+Now we will explore both options in more detail.
```
Comment (joel-hamill) [inline 1.9/usage/tutorials/dcos-101/loadbalancing.md]: Delete/replace this line with suggestion above.

File: 1.9/usage/tutorials/dcos-101/loadbalancing.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+* First, scale app2 to two instances: `dcos marathon app update /dcos-101/app2 instances=2`
+* Marathon-lb
+    * Check app2 as before via `http://<public-node>10000`. When you do this repeatedly you should see the request being served by different instances of app2.
+    * You can also check the Marathon-LB stats via `http://<public-node>:9090/haproxy?stats`
+* Named VIPs
+    * We have seen named vips before for service discovery inside our cluster
```
Comment (joel-hamill) [inline 1.9/usage/tutorials/dcos-101/loadbalancing.md]: delete line 23

File: 1.9/usage/tutorials/dcos-101/loadbalancing.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+    * SSH to the leading master node: `dcos node ssh --master-proxy --leader`
+    * `curl dcos-101app2.marathon.l4lb.thisdcos.directory:10000`. When you do this repeatedly you should see the request being served by different instances.
+* Scale app2 back to one instance: `dcos marathon app update /dcos-101/app2 instances=1`
+
+
+
+# Outcome
+We used two different methods to load balance requests for two different instances of our app.
```
Comment (joel-hamill) [inline 1.9/usage/tutorials/dcos-101/loadbalancing.md]: `We used Marathon-LB and VIPs to...`

File: 1.9/usage/tutorials/dcos-101/loadbalancing.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+
+# Deep Dive
+The two load balancing options in DC/OS offer different advantages and drawbacks that you should take into account when choosing one over the other
```
Comment (joel-hamill) [inline 1.9/usage/tutorials/dcos-101/loadbalancing.md]: Missing period

Comment (joel-hamill) [inline 1.9/usage/tutorials/dcos-101/loadbalancing.md]: `The two load balancing options in DC/OS offer different features and benefits that you should consider when choosing or the other.`

File: 1.9/usage/tutorials/dcos-101/marathon-lb.md [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+
+  * Install Marathon-LB: `dcos package install marathon-lb`
+  * Check that it is running: `dcos task` and identify the IP adress of the public agent node (Host) where Marathon-LB is running,
+    * Warning: If you started your cluster using a cloud provider (especially AWS) dcos task might show you the private ip address of the host, which is not resolvable from outside the cluster (e.g., if you see something like 10.0.4.8 it is very likely a private address).
+    In that case, you need to retrieve the public IP from your cloud provider. On AWS, go to the console and then search for the instance with the private IP shown by 'dcos task'. The public IP will be listed in the instance description as Public IP.
+  * Connect to the webapp (from your local machine) via `<Public IP>:10000`. You should see a rendered version of the web page including the physical node and port app2 is running on.
+  * Use the web form to add a new Key:Value pair
+  * You can verify the new key was added in two ways:
+    1. Check the total number of keys using app1: `dcos task log app1`
+    2. Check redis directly
```
Comment (joel-hamill) [inline 1.9/usage/tutorials/dcos-101/marathon-lb.md]: Add period

File: 1.9/usage/tutorials/dcos-101/redis-package.md [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+post_title: Install First Package
+nav_title: Installing First Package
+menu_order: 2
+---
+
+Welcome to part 2 of the DC/OS 101 Tutorial.
+
+# Prerequisites
+By now, you have a running DC/OS cluster and the DC/OS CLI installed and configured. If that isn't the case, please follow these [instructions](/docs/1.9/usage/tutorials/dcos-101/cli/).
+Furthermore, we use [jq](https://stedolan.github.io/jq/) as a json processor to simplify some of the commands below.
```
Comment (joel-hamill) [inline 1.9/usage/tutorials/dcos-101/redis-package.md]: `We use [jq]...`

File: 1.9/usage/tutorials/dcos-101/resources.md [hunk 14]
```diff
@@ -0,0 +1,98 @@
...
+nav_title: Understanding Resources
+menu_order: 8
+---
+
+# Prerequisites
+* A [running DC/OS cluster](/docs/1.9/usage/tutorials/dcos-101/cli/) with [the DC/OS CLI installed](/docs/1.9/usage/tutorials/dcos-101/cli/).
+* [app2](/docs/1.9/usage/tutorials/dcos-101/app2/) deployed and running in your cluster.
+
+# Objective
+In this section we will learn how to monitor and understand our resource utilization, how resource limits are enforced, and how to debug resource management issues.
```
Comment (joel-hamill) [inline 1.9/usage/tutorials/dcos-101/resources.md]: `In this section you will learn...` In general you want to address the readers in the first person - not as `we`.

File: 1.9/usage/tutorials/dcos-101/resources.md [hunk 15]
```diff
@@ -0,0 +1,98 @@
...
+  "env": null,
+  "instances": 1,
+  "cpus": 1,
+  "mem": 128,
+  "disk": 0,
+  "gpus": 0,
+  ...
+```
+
+  *Note:* The cpus, men, disk, and gpus parameters above specify the allocated resources and therefore define the maximum amount of resources a task can use. This number is not necessarily the same as the amount of resources a task actually uses. That number is usually lower.
```
Comment (joel-hamill) [inline 1.9/usage/tutorials/dcos-101/resources.md]: `CPUs`

Comment (joel-hamill) [inline 1.9/usage/tutorials/dcos-101/resources.md]: `GPUs`

File: 1.9/usage/tutorials/dcos-101/service-discovery.md [hunk 16]
```diff
@@ -0,0 +1,72 @@
...
+# Prerequisites
+* A [running DC/OS cluster](/docs/1.9/usage/tutorials/dcos-101/cli/) with [the DC/OS CLI installed](/docs/1.9/usage/tutorials/dcos-101/cli/).
+* [app1](/docs/1.9/usage/tutorials/dcos-101/app1/) deployed and running in your cluster.
+
+# Objective
+Our application in the previous part used `redis.marathon.l4lb.thisdcos.directory:6379` as the address for connecting to redis. As redis might be running on any agent in the cluster (and furthermore on different ports), how does this address link to the actual running redis instance?
+In this section, we will learn about DC/OS service discovery by exploring the different options for service discovery for apps in DC/OS.
+
+# Steps
+  [Service discovery](https://dcos.io/docs/1.9/usage/service-discovery/) allows you to connect to your applications without necessarily knowing where they are running. Service discovery is particularly useful in cases where applications may fail and be restarted on a different host.
```
Comment (joel-hamill) [inline 1.9/usage/tutorials/dcos-101/service-discovery.md]: links should be relative, e.g. `(/docs/1.9/usage/service-discovery/)`

File: 1.9/usage/tutorials/dcos-101/service-discovery.md [hunk 17]
```diff
@@ -0,0 +1,72 @@
...
+
+  DC/OS provides two options for service discovery: Mesos-DNS and Named virtual IPs.
+  * SSH into your cluster to see how these service discovery methods work: `dcos node ssh --master-proxy --leader`,
+  <a name="mesosdns"></a>
+  * [Mesos-DNS](/docs/1.9/usage/service-discovery/mesos-dns/) assigns a Mesos-DNS for every Marathon app. The naming pattern is  *task.scheduler.mesos* and the default scheduler for jobs is `marathon`, so the Mesos-DNS name for our redis service is *redis.marathon.mesos*.
+
+  Let us use [dig](https://linux.die.net/man/1/dig) to retrieve the address record (also called the A record): `dig redis.marathon.mesos`.
```
Comment (joel-hamill) [inline 1.9/usage/tutorials/dcos-101/service-discovery.md]: `Let's use...`

**Comment by joel-hamill** [comment]
@joerg84 ping ^

**Pull request merged**